R/descriptives.R

```diff
@@ -270,3 +311,70 @@ mean_sd_func <- function(df, cont_vars, treatment, weights) {
 
   }
 }
+
+#' Creates summary table for continuous variables with median/IQR/Kruskal-Wallis.
+#'
+#' @inheritParams cont_func
+#'
+#' @import dplyr
+#' @import tidyr
+#' @import purrr
+#'
+#' @noRd
+median_IQR_func <- function(df, cont_vars, treatment) {
```

Comment:
I'd consider separating the helpers into separate function scripts.  They could organized based on relatedness, like:

- `pvalues.R`: all helper functions that calculate p-values
- `weighted.R`: `weighted_mean`, `weighted_quantile`, etc.
- `check_types.R`: `is_numeric_variable`, `is_categorical_variable`

We could also rename these to be  `median_IQR_descriptives`, `mean_sd_descriptives`, `cont_descriptives`and `cat_descriptives` to make it clearer they are what is underneath the top-level function.  They make sense leaving in the `descriptives.R` file